vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_24.py

```diff
@@ -112,6 +156,13 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
         :param layer: The layer with the weights to be processed
         
         """
+        if self.do_sparse_decompress:
+            layer.weight.data = self._decompress_bitmask_compressed_weight(
+                compressed=layer.compressed,
+                bitmask=layer.bitmask,
```

Comment:
Should we delete `layer.compressed` and `layer.bitmask` after decompressing them?

Reply:
We absolutely can, addressed in latest commit!